aggregates/src/commonMain/kotlin/org/dhis2/mobile/aggregates/ui/DataSetTableScreen.kt

```diff
@@ -569,6 +615,36 @@ private fun DataSetTableContent(
                                 dataSetDetails = dataSetDetails,
                             )
 
+                            if (dataSetSectionTable.tables().isEmpty()) {
```

Comment:
isn't this code duplicated?. Why don't you create a private composable function to reuse the code?